### PR TITLE
Add PyMysql component

### PR DESCRIPTION
**Pull request by alonelaval**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- 
- [x]  New feature provided

- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: oap-server/server-bootstrap/src/main/resources/component-libraries.yml [hunk 1]
```diff
@@ -416,6 +416,9 @@ Flask:
 Requests:
   id: 7002
   languages: Python
+PyMysql:
```
Comment (wu-sheng) [inline oap-server/server-bootstrap/src/main/resources/component-libraries.yml:419]: I think this is MySQL driver in Python, we need a mapping for this to show MySQL server icon in the topology, 

Ref to, https://github.com/apache/skywalking-rocketbot-ui/pull/331#issuecomment-660499481

Comment (alonelaval) [reply oap-server/server-bootstrap/src/main/resources/component-libraries.yml:419]: OK

**Comment by alonelaval** [comment]
> As I mentioned in the PR [apache/skywalking-python#35 (review)](https://github.com/apache/skywalking-python/pull/35#pullrequestreview-448633625) we need a mapping this time, @alonelaval please add one, refer to line 449 in this file, and thus we don't need a logo then

ok.

File: oap-server/server-bootstrap/src/main/resources/component-libraries.yml [hunk 2]
```diff
@@ -480,3 +483,4 @@ Component-Server-Mappings:
   mariadb-jdbc: Mariadb
   Mysqli: Mysql
   Predis: Redis
+  PyMsql: Mysql
```
Comment (kezhenxu94) [inline oap-server/server-bootstrap/src/main/resources/component-libraries.yml:486]: typo? PyMysql

Comment (alonelaval) [reply oap-server/server-bootstrap/src/main/resources/component-libraries.yml]: > typo? PyMysql

PyMysql

**Pull request merged**